spec/unit/shoulda/matchers/active_model/validate_numericality_of_matcher_spec.rb

```diff
@@ -1157,6 +1247,10 @@ def validate_numericality
     validate_numericality_of(attribute_name)
   end
 
+  def validate_virtual_attribute_numericality
```

Comment:
You can probably remove this too, right? Since this is now the same as `validate_numericality`.